www/pages/mergin-web.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Mergin website
+permalink: /mergin-web
+---
+
+Mergin website provides a portal to register, manage and share your projects. Documentations below provide help navigating the website.
+
+### Index
+
+- [Registration](registration) - registration and signing up to the Mergin service
+- [Dashboard](dashboard) - accessing to your recent projects, profile and other settings
+- [Creating projects](web-create-project) - how to create new project and upload files
+- [Subscriptions](subscriptions) - How to subscribe to Mergin Plans
+- [Project details](project-details) - view project files, history and changelogs
+- [Sharing projects](sharing-project) - share your projects with other Mergin users

```

Comment:
the `sharing-project` page is not modified in this PR, but:

- it contains URL that is not a link (can't be clicked): https://public.cloudmergin.com/projects/saber/zoomstack
- it has a screenshot that's outdated:

![image](https://user-images.githubusercontent.com/193367/111200101-a9453700-85c1-11eb-876a-be23fdd4366e.png)

Reply:
The link is intentional, so users will see what link to send.